sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Release History
 
-## 1.7.0-beta.1 (Unreleased)
+## 1.7.0 (2020-12-14)
 
 ### Added
 - `System.Text.Json.JsonConverter` implementation for the `ETag`
+- Synchronous `HttpClient` support on .NET 5.
 
-### Fixes
+### Fixed
```

Comment:
I believe this should be "Key Bug Fixes" according to the [guidelines](https://azure.github.io/azure-sdk/policies_releasenotes.html)